client/my-sites/checkout/checkout/credit-card-selector.jsx

```diff
@@ -36,22 +33,8 @@ class CreditCardSelector extends React.Component {
 	}
 
 	storedCards = () => {
-		return this.props.cards.map( ( card ) => {
-			const onSelect = () => this.handleClickedSection( card.stored_details_id );
-			return (
-				<CreditCard
-					key={ card.stored_details_id }
-					className="checkout__payment-box-section"
-					card={ {
-						lastDigits: card.card,
-						cardType: card.card_type,
-						name: card.name,
-						expiry: card.expiry,
-					} }
-					selected={ card.stored_details_id === this.state.section }
-					onSelect={ onSelect }
-				/>
-			);
+		return this.props.cards.map( () => {
```

Comment:
This is great! You could go even further here and remove the `storedCards` method (and wherever it is called), since it does nothing.

Reply:
Good idea. I've cleaned it up!